src/utils.ts

```diff
@@ -23,28 +30,32 @@ export function wait(n: number): Promise<void> {
 	return new Promise((resolve) => setTimeout(resolve, n));
 }
 
-export function getTag(name: string): string {
+export function getTag(name: string): MediaType {
 	return EP_REGEX.test(name)
-		? "episode"
+		? MediaType.EPISODE
 		: SEASON_REGEX.test(name)
-		? "pack"
+		? MediaType.SEASON
 		: MOVIE_REGEX.test(name)
-		? "movie"
-		: "unknown";
+		? MediaType.MOVIE
+		: MediaType.OTHER;
 }
 
 export type Result<T> = T | Error;
 
 export const ok = <T>(r: Result<T>): r is T => !(r instanceof Error);
 
+export function cleanseSeparators(str: string): string {
+	return str
+		.replace(/[._()[\]]/g, " ")
```

Comment:
missing an escape for the [ here?

should be `/[._()\[\]/g` no?

Reply:
`[` doesn't need to be escaped inside a `[]` but `]` does